src/Bundle/DependencyInjection/Compiler/RegisterStateMachinePass.php

```diff
@@ -82,6 +85,28 @@ private function setWinzouAsStateMachine(ContainerBuilder $container): void
         $stateMachineDefinition->addArgument(new Reference('sm.factory'));
     }
 
+    private function registerWinzouStateMachine(ContainerBuilder $container): void
+    {
+        if (!$this->isWinzouStateMachineEnabled($container)) {
+            return;
+        }
+
+        $stateMachineDefinition = $container->register('sylius.resource_controller.state_machine.winzou', StateMachine::class);
+        $stateMachineDefinition->setPublic(false);
+        $stateMachineDefinition->addArgument(new Reference('sm.factory'));
+    }
+
+    private function registerSymfonyWorkflowStateMachine(ContainerBuilder $container): void
+    {
+        if (!$this->isSymfonyWorkflowEnabled($container)) {
+            return;
+        }
+
+        $stateMachineDefinition = $container->register('sylius.resource_controller.state_machine.symfony', Workflow::class);
```

Comment:
Won't workflow be a better naming? 
```suggestion
        $stateMachineDefinition = $container->register('sylius.resource_controller.state_machine.workflow', Workflow::class);
```

Reply:
We name it symfony cause we name winzou the other one and WE don't know if a future component from another owner could be named workflow too